ext/php7/logging.h

```diff
@@ -4,10 +4,7 @@
 
 #include "configuration.h"
 
-inline void ddtrace_log_err(const char *message) {
-    TSRMLS_FETCH();
-    php_log_err((char *)message TSRMLS_CC);
-}
+inline void ddtrace_log_err(const char *message) { php_log_err((char *)message); }
```

Comment:
wtf is clang-format converting this to a single line?!?

Reply:
Yes. It bothered me at first too, but now it's my preference in all languages! Trivial things taking up trivial amounts of space on my screen is nice.